### ISSUE TITLE
[highlightElement] Typescript get Error does not exist on type 'HLJSApi'.

### ISSUE THREAD

**Issue by mtr1990** [highlightjs/highlight.js#3082]
Hi,

I encountered this problem when implementing in typescript ("^4.2.3")

```
function CodeBlock({ value }: CodeBlockProps) {
  useEffect(() => {
    document
      .querySelectorAll<HTMLElement>('pre > code')
      .forEach((block: HTMLElement) => {
        // hljs.highlightBlock(block); 
        => work but get Info
        - Deprecated as of 10.7.0. highlightBlock will be removed entirely in v12.0
        - Deprecated as of 10.7.0. Please use highlightElement now.
        
        hljs.highlightElement(block);  
        => Not work I'm using "highlight.js": "^10.7.1"
      });
  }, [value]);

  return (
    <pre dir="ltr">
      <code>{value}</code>
    </pre>
  );
}
```

```
Property 'highlightElement' does not exist on type 'HLJSApi'.  TS2339

    110 |       .forEach((block: HTMLElement) => {
    111 |         // hljs.highlightBlock(block);
  > 112 |         hljs.highlightElement(block);
        |              ^
    113 |       });
    114 |   }, [value]);
    115 |

```

How can i fix this problem, thank you!

**Comment by joshgoebel** [comment]
Fixed 7880cba8.

You should be able to use @ts-ignore for now to ignore the error, it's just an issue with our types file.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments

**Comment by mtr1990** [comment]
Thanks!
